hack/benchmark/e2e/gateway/vald/vald_bench_test.go

```diff
@@ -55,63 +53,17 @@ func init() {
 	wait = time.Duration(time.Duration(waitSeconds) * time.Second)
 }
 
-func BenchmarkValdGateway_REST_Sequential(b *testing.B) {
-	ctx := context.Background()
-
-	for _, name := range targets {
-		bench := e2e.New(
-			b,
-			e2e.WithName(name),
-			e2e.WithClient(
-				rest.New(
-					rest.WithAddr(
-						restAddr,
-					),
-				),
-			),
-			e2e.WithStrategy(
-				strategy.NewInsert(),
-				strategy.NewSearch(),
-			),
-		)
-		bench.Run(ctx, b)
-	}
-}
-
-func BenchmarkValdGateway_REST_Stream(b *testing.B) {
-	ctx := context.Background()
-
-	for _, name := range targets {
-		bench := e2e.New(
-			b,
-			e2e.WithName(name),
-			e2e.WithClient(
-				rest.New(
-					rest.WithAddr(
-						restAddr,
-					),
-				),
-			),
-			e2e.WithStrategy(
-				strategy.NewStreamInsert(),
-				strategy.NewStreamSearch(),
-			),
-		)
-		bench.Run(ctx, b)
-	}
-}
-
 func BenchmarkValdGateway_gRPC_Sequential(b *testing.B) {
-	ctx := context.Background()
-	client, err := grpc.New(ctx,
-		grpc.WithAddr(
+	client := vald.New(
+		vald.WithClient(grpc.New(
+			grpc.WithAddrs(grpcAddr),
+		)),
+		vald.WithAddr(
```

Comment:
what is the difference between the `addr` and `client.Addr`?

Reply:
????

Reply:
why do we need to set the addr twice?

Reply:
one for grpc.Client one for vald.RPC client

Reply:
thank you!